amqp-impl/src/main/java/io/streamnative/pulsar/handlers/amqp/ExchangeContainer.java

```diff
@@ -27,24 +29,20 @@
 import org.apache.pulsar.common.naming.NamespaceName;
 import org.apache.qpid.server.exchange.ExchangeDefaults;
 
-
 /**
  * Container for all exchanges in the broker.
  */
 @Slf4j
 public class ExchangeContainer {
 
     private static final Map<String, AmqpExchange.Type> BUILDIN_EXCHANGE_NAME_SET = new HashMap<>();
-    private static PulsarService pulsarService;
+    private static Map<NamespaceName, CountDownLatch> countDownLatchMap = new HashMap<>();
```

Comment:
One countdown latch is enough, we don't need to introduce a map here.

Reply:
Each Vhost has its own default exchanges, only one countdownLatch can not work well in the condition of creating multiple Vhost at the same time.

Reply:
Is the countdown latch only used when the broker startup? or namespace bundle loaded? If just the broker startup, I think you can get how many vhosts. If on the namespace bundle loaded or client trigger load default exchanges, we should avoid introducing a block operation here.

Reply:
It is used on the namespace bundle loaded, if don't add this, should we need to handle this problem？